### DISCUSSION TITLE
Shipping Confirmation: Update shipping and tracking information for the orders

### DISCUSSION THREAD

**Discussion by hardikradixweb** [amzn/selling-partner-api-models#267] [Q&A] [answered]
Hello,

I have successfully implemented the orders pull system where I can pull orders and order items, but now I have to update shipping information to those orders I have tried to understand the documentation but there are many shipping APIs that amazon provides, so do anyone have any idea about how I can update shipping information (USPS, FedEx, UPS) to the amazon orders? and by which API i can achieve this?

Thanks in the Advance!!

**Comment by doraemonjcl** [comment, marked as answer]
You could use confirmShipment operation from Orders API.  You can also use Feeds API to upload batch shipments for multiple orders. But for single order, the newly released confirmShipment operation is suitable. https://developer-docs.amazon.com/sp-api/docs/orders-api-v0-reference#post-ordersv0ordersorderidshipmentconfirmation

**Comment by ritikacd** [comment]
We also have an amazing blog on confirming shipments using SP-API with examples: https://developer-docs.amazon.com/sp-api-blog/docs/how-to-confirm-mfn-orders-using-selling-partner-api

**Reply by ericblade** [reply]
hey, so, this blog should have a RSS feed available so we can subscribe to it and get notifications of new posts, and such.  Probably some decent information in here.

**Comment by michabbb** [comment]
hi all, how do i mark an order as "delivered" ? (a regular order the seller ships, so I am not talking about fulfillment, ISPU, or Easy Shipping). thanks for any hint !

**Comment by doraemonjcl** [comment]
Hi Michael,

**Comment by doraemonjcl** [comment]
I guess you mean marking the orders as "Shipped". Please use confirmShipment operation of Orders API (for your own shipping label purchased from carriers) or use Merchant
Fulfillment API (for buy shipping) to mark a seller-fulfilled order as shipped.
Best,
Jacob Liang

**Reply by michabbb** [reply]
thanks for the feedback. i was able to figure out that there is no status "delivered" 😏 🙈 
(at least for regular shippings the seller is doing on its own with a carrier amazon supports, like DPD, UPS....)
